Library/Homebrew/cmd/config.rb

```diff
@@ -119,8 +119,10 @@ def describe_system_ruby
     s
   end
 
-  def java_version
-    java = `java -version 2>&1`.lines.first.chomp
+  def describe_java
+    java = which "java"
+    return "N/A" if java.nil?
+    java = `#{java} -version 2>&1`.lines.first.chomp
```

Comment:
I don't know if it's particularly a problem, and it only happens on first call to Java after system reboot/awake/etc, but with Oracle's Java calling ` /usr/libexec/java_home --exec java -version ` takes roughly 30 seconds to respond. This might knock onto ` brew config ` slightly. I don't think it's particularly an issue given it only happens in that limited circumstance.